README.md

```diff
@@ -50,6 +50,8 @@ model_name = "average_word_embeddings_komninos"
 # model_name = "sentence-transformers/all-MiniLM-L6-v2"
 
 model = SentenceTransformer(model_name)
+# or directly from mteb:
+model = mteb.get_model(model_name)
```

Comment:
Should we just recommend this one always? 

and maybe rephrase to:
```suggestion
model = mteb.get_model(model_name) # if the model is not implemented in MTEB it will be eq. to SentenceTransformer(model_name)
```

Reply:
Yes, but I'm not sure removing it is the best approach. I can make changes to whatever you think is better